k8s/jenkins/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,37 @@
 actions:
-- name: dummy test
+- name: Jenkins service login
   bashTest:
     script: |-
-      exit 0
+      SERVICE="${APP_INSTANCE_NAME}-jenkins-ui"
+      IP=''
+      PORT=''
+      PASSWORD=''
+
+      IP=$(kubectl get service "${SERVICE}" \
```

Comment:
We could actually move the variables assignments from this file to tester.sh, so they would be available in all tests in this file. Then we could split the negative and positive login tests into two separate ones to increase the readability and simplify verification in case of failure.